openshift4/templates/redis.dc.yaml

```diff
@@ -14,6 +14,12 @@ metadata:
     openshift.io/provider-display-name: Red Hat, Inc.
     openshift.io/documentation-url: https://github.com/sclorg/redis-container/tree/master/3.2
     openshift.io/support-url: https://access.redhat.com
+parameters:
```

Comment:
@yasserhu should this one be here too? I can copy the file from dev and remove it if not